models/news_model.js

```diff
@@ -0,0 +1,9 @@
+
+const db = require("../db/index.js");
+
+exports.fetchTopics = (()=>{
+  return db.query('SELECT * FROM topics;')
+  .then((topics) =>{
```

Comment:
the thing that comes back from the request is not the topics. Be careful of your variable names.

Reply:
Is this something i should do for all the places where i have called topics? or just here in the model?